src/_includes/meta.html

```diff
@@ -1,17 +1,43 @@
-{% if include.title %} {% assign title=include.title|prepend=" - " %} {% else %} {% assign title="" %} {% endif %}
+{%- if page.title -%}
+  {%- assign page_title = page.title -%}
+  {%- assign title = page.title | append: " - " | append: site.title -%}
+{%- else -%}
+  {%- assign page_title = site.title -%}
+  {%- assign title = site.title -%}
+{%- endif -%}
+
+{%- if page.description -%}
+  {%- assign description = page.description | strip_newlines -%}
```

Comment:
You might've already seen this same comment in Slack, but putting on the PR too for good measure:

The meta description for the recent ODS press release at least seems to lose spacing due to this `strip_newlines`

<img src="https://github.com/cal-itp/calitp.org/assets/25497886/7481018a-54b1-44da-b0d8-5b80a7d7af45" width="400">

`<meta name="description"...>`
<img src="https://github.com/cal-itp/calitp.org/assets/25497886/cc23a154-cf15-4ad0-a675-2a849f8089ae" width="600">